### PR TITLE
TRUNK-5955 Modifiers should be declared in the correct order

### PR DESCRIPTION
**Pull request by insookwa**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5955

## Checklist: I completed these to help reviewers :)

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/context/Context.java [hunk 1]
```diff
@@ -185,7 +185,7 @@ public static void setDAO(ContextDAO dao) {
 	/**
 	 * Spring init method that sets the authentication scheme.
 	 */
-	static private void setAuthenticationScheme() {
+    private static  void setAuthenticationScheme() {
```
Comment (sacull) [inline api/src/main/java/org/openmrs/api/context/Context.java:188]: You put four spaces here, and there should be one tab. ;)

File: api/src/main/java/org/openmrs/util/MemoryAppender.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
 	// we store the MemoryAppenders by name, using SoftReferences to allow them to be garbage collected
 	// as an implementation detail, we expect this class to only have a single instance, so our map
 	// is only allocated an initial capacity of 1
-	private final static Map<String, SoftReference<MemoryAppender>> APPENDERS = new HashMap<>(1);
+private static final  Map<String, SoftReference<MemoryAppender>> APPENDERS = new HashMap<>(1);
```
Comment (sacull) [inline api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: There is also no tab here, in addition, there are probably two spaces between the words "final" and "Map". ;)

**Review by sacull — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/util/MemoryAppender.java (hunk 2 above)
Comment (achilep) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: > There is also no tab here, in addition, there are probably two spaces between the words "final" and "Map". ;)

hello @insookwa , 
have you seen these comments?

Comment (achilep) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: please @insookwa  can you squash your commit? 
Thanks

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: @achilep did you get a chance to look at this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

Comment (insookwa) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: thanks @dkayiwa  i am still finding ways of squashing commits . should I just delete this pull request and commit another one with the proper changes.

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: Did you read the link that i shared?

Comment (insookwa) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: Actually the link discourages squashing which I've been trying to figure out. I am very sorry   . is there a way I can fix this best @dkayiwa

File: api/src/main/java/org/openmrs/api/context/Context.java [hunk 3]
```diff
@@ -185,7 +185,7 @@ public static void setDAO(ContextDAO dao) {
 	/**
 	 * Spring init method that sets the authentication scheme.
 	 */
-	static private void setAuthenticationScheme() {
+	private static  void setAuthenticationScheme() {
```
Comment (sacull) [inline api/src/main/java/org/openmrs/api/context/Context.java:188]: You've fixed the spaces at the beginning of the line nicely, but I suggest you turn on whitespace visibility in your ide.

image
Do you see the two dots between the words "static" and "void"? One dot is one space, and there are two. ;)

File: api/src/main/java/org/openmrs/util/MemoryAppender.java (hunk 2 above)
Comment (sacull) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: image
Here, the tab at the beginning is still missing and there are two spaces between the words "final" and "Map". ;)

Comment (sacull) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: If we take several adjacent lines, the lack of tabs will be better seen:
image

**Review by sacull — CHANGES_REQUESTED** [review]

**Comment by insookwa** [comment]
thanks, @sacull I've made the changes. now how do I submit the changes without squashing and following the required OpenMRS procedure

**Comment by sacull** [comment]
Without squashing? I'm sorry but I do not know. Personally, I do squash followed by a push with the force option. It's not recommended as it deletes history, but they stuck it into my head two months ago that this should be done here, and I still do. ;)

**Comment by insookwa** [comment]
@dkayiwa  how can i submit the changes without squashing commits . i am done with the changes and we can close this pill request

File: api/src/main/java/org/openmrs/util/MemoryAppender.java (hunk 2 above)
Comment (achilep) [reply api/src/main/java/org/openmrs/util/MemoryAppender.java:47]: thanks @dkayiwa  for this information .

> @achilep did you get a chance to look at this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

**Comment by achilep** [comment]
https://github.com/openmrs/openmrs-core/pull/3551/commits 
I think it's Ok like that. @sacull  has only to approve the pull request .

**Comment by sacull** [comment]
If there is a commit, it will be approved. For now, I don't see a commit with the changes I wrote about earlier. :P

**Comment by sacull** [comment]
@insookwa What have you changed now? I don't see any difference from the previous commit.

**Review by sacull — APPROVED** [review]
It's very good now.

As for Travis, I started the tests locally and they all passed, so they probably pass for you. Travis must be capricious then. ;)

**Pull request merged**